CHANGELOG.md

```diff
@@ -1,7 +1,22 @@
 # Change Log
 All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org)
 
-## 3.1.6-dev
+## 3.2.0 - 2023-05-22
+
+### Added
+
+- New env:rebuild-code command (#2417)
+- Validation for aliases command (#2447)
+
+### Fixed
+- Bug with multidev:delete when ids are numbers-only (#2462)
+
+### Changed
+- Add upstream label field to site:info (#2455)
+- Do not check for user membership to remove supporting org (#2453)
+- Improve error messages on error 5xx (#2454)
+- Add search and replace to env:clone-content command for WordPress sites (#2448)
+- Deprecate getSite with param (#2449)
```

Comment:
```suggestion
### Added

- New env:rebuild-code command ([#2417](https://github.com/pantheon-systems/terminus/pull/2417))
- Validation for aliases command ([#2447](https://github.com/pantheon-systems/terminus/pull/2447))

### Fixed
- Bug with multidev:delete when ids are numbers-only ([#2462](https://github.com/pantheon-systems/terminus/pull/2462))

### Changed
- Add upstream label field to site:info ([#2455](https://github.com/pantheon-systems/terminus/pull/2455))
- Do not check for user membership to remove supporting org ([#2453](https://github.com/pantheon-systems/terminus/pull/2453))
- Improve error messages on error 5xx ([#2454](https://github.com/pantheon-systems/terminus/pull/2454))
- Add search and replace to env:clone-content command for WordPress sites ([#2448](https://github.com/pantheon-systems/terminus/pull/2448))
- Deprecate getSite with param ([#2449](https://github.com/pantheon-systems/terminus/pull/2449))
```

Not sure if github was just acting up in the PR view, but issue IDs did not auto-link when I previewed just now.

Reply:
I hate to say "we have always done it this way" but "we have always done it this way" 😂  and I'm not sure if there is a reason (maybe keep it compact?)

@greg-1-anderson any thoughts?

Reply:
Did not make this a "changes requested" for that reason 😁 .